### PR TITLE
Change contactUsHero

### PR DESCRIPTION
**Pull request by Mariii-23**
# Description
The main goal of this pr is to change the ContactUsHero and the contact's page.
image
image

The size and colors of the font will be perfected in another PR.

## Issue

https://github.com/b1tsway/b1tsway.github.io/issues/65

## Type of change

Please delete options that are not relevant or select with a "x" the correct option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist(mark with a "x" the items you made):

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I tested all related functionalities and I didn´t break anything

### CODE REVIEW

File: src/components/molecules/Hero/ContactUsHero/index.tsx [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-            <h1>The Perfect Development Team For Your Needs</h1>
-            <div className="description">
-                <h3>
-                    Let us transform your vision into reality
-                    through our expert software development services.
-                </h3>
-            </div>
+        <div>
+            <h1>Have a challenge for us? </h1>
+            <h1 className="text-lines">Let's Talk!!!</h1>
```
Comment (assebc) [inline src/components/molecules/Hero/ContactUsHero/index.tsx:9]: dont like this effect

File: src/components/molecules/Hero/ContactUsHero/styles.css [hunk 2]
```diff
@@ -7,49 +7,46 @@
...
+.text-lines::before {
+  content: "";
+  display: block;
+  width: 70%;
+  height: 4px; 
+  left: 32px;
+  background-color:  #008f8f;
+  position: absolute;
+  position: absolute;
+  bottom: -16px;
```
Comment (assebc) [inline src/components/molecules/Hero/ContactUsHero/styles.css:51]: this effect

File: src/pages/ContactUs/index.tsx [hunk 3]
```diff
@@ -12,7 +12,12 @@ export const ContactUs: FC = () => {
             <Hero.ContactUs />
             <div className="contact-wrapper">
                 <Layout.Body>
-                    <h1>{"WE LOOK FORWARD TO HEARING ABOUT YOUR PROJECT."}</h1>
+                    <div className="text-form-container">
+                        <h1>{"Tell us about your project"}</h1>
+                        <p>Try to provide us with the most relevant information about you and your project.
```
Comment (assebc) [inline src/pages/ContactUs/index.tsx:17]: "Enrich us about your projects and ideas"

**Review by assebc — CHANGES_REQUESTED** [review]

File: src/components/molecules/Hero/ContactUsHero/index.tsx (hunk 1 above)
Comment (Mariii-23) [reply src/components/molecules/Hero/ContactUsHero/index.tsx]: Fix. I remove the effect

File: src/components/molecules/Hero/ContactUsHero/styles.css (hunk 2 above)
Comment (Mariii-23) [reply src/components/molecules/Hero/ContactUsHero/styles.css]: what do you think know?

image

File: src/pages/ContactUs/index.tsx (hunk 3 above)
Comment (Mariii-23) [reply src/pages/ContactUs/index.tsx]: I like this:
image
What do you think?

**Review by assebc — CHANGES_REQUESTED** [review]
didnt like how it turns out im sorry!

try :

"Let us transform your vision into reality through our expert software development services."

as text before form

inside hero, remove let's talk and put the "enrich..." text

**Comment by Mariii-23** [comment]
> 
image

File: src/components/molecules/Hero/ContactUsHero/index.tsx [hunk 4]
```diff
@@ -4,14 +4,8 @@ import "./styles.css"
...
-        <div>
-            <h1>Have a challenge for us? </h1>
-            <h1>Let's Talk!!!</h1>
-        </div>
-        <h3>
-            Let us transform your vision into reality
-            through our expert software development services.
-        </h3>
+        <h1>Have a challenge for us? </h1>
+        <h1>{"Enrich us about your projects and ideas"}</h1>
```
Comment (assebc) [inline src/components/molecules/Hero/ContactUsHero/index.tsx:8]: why this tipography have  different signatures but are the same, uniformize it!

**Review by assebc — CHANGES_REQUESTED** [review]

File: src/components/molecules/Hero/ContactUsHero/index.tsx (hunk 4 above)
Comment (Mariii-23) [reply src/components/molecules/Hero/ContactUsHero/index.tsx]: Done

**Review by assebc — APPROVED** [review]
lgtm!

**Pull request merged**